### PR TITLE
Add getVolume method

### PR DESCRIPTION
**Pull request by dayvson**

### CODE REVIEW

File: src/components/media_control/media_control.js [hunk 1]
```diff
@@ -131,6 +132,11 @@ export default class MediaControl extends UIObject {
     this.container.stop()
   }
 
+  onVolumeChanged(event) {
+    this.mute = this.currentVolume === 0
```
Comment (flavioribeiro) [inline src/components/media_control/media_control.js]: mind adding a `(this.currentVolume === 0)`?

**Comment by flavioribeiro** [comment]
LGTM, also thanks for the styling refactory

**Comment by dayvson** [comment]
Can you merge it?

**Comment by dayvson** [comment]
I also added the PLAYER_VOLUMEUPDATE event

**Comment by leandromoreira** [comment]
I think it should @tjenkinson (handle it as a different thing)

**Pull request merged**